### PR TITLE
Adding general little things

### LINKED ISSUE #7: General little things
1. Need a command argument for add olu-status tar files to the olu-status cache. Something like: `python3 -m updater_maker --add star-tracker_olu-status_12345.tar.gz`
2. Add a option to move generate updates to the update cache after calling `make` in the cli

### PR DESCRIPTION
**Pull request by mairacanal**
Closes #7 

Just adding the command argument for add olu-status tar files to the olu-status cache and the option to move generate updates to the update cache after calling make in the cli.

### CODE REVIEW

**Review by ryanpdx — CHANGES_REQUESTED** [review]
Looks good.

Just a quick change as I don't think the add cache functionality belongs in the `UpdateMaker` class.

Also since this is now using `ArgumentParser` could you change it so `usage()` is only the cli usage message and is only called when `help` is entered in the CLI. Then `ArguementParser` can print out its program help message.

File: update_maker/update_maker.py [hunk 1]
```diff
@@ -71,6 +72,18 @@ def __init__(self, board: str):
             if i.endswith(".deb"):
                 remove(DOWNLOAD_DIR + i)
 
+        # add olu-status tar files to the olu-status cache
```
[the remark this answers is not in the document]
Comment (mairacanal) [reply update_maker/update_maker.py:75]: I did this to make it possible the command `python3 -m update_maker generic --add ./tests/test_files/generic_olu-status_1614805451.tar.xz`. If I put this outside of the `UpdateMaker` class, it won't be possible to run this command the first time. You will have to run the following commands:

`
python3 -m update_maker --add ./tests/test_files/generic_olu-status_1614805451.tar.xz
python3 -m update_maker generic
`

So what do you think is better?

[the remark this answers is not in the document]
Comment (ryanpdx) [reply update_maker/update_maker.py:75]: Or as a member in the `UpdateMaker` class. Just not in `__init__()`.

**Review by ryanpdx — APPROVED** [review]
Looks good!

**Pull request merged**